plugins/aws/autoscaling/webTierAsgCloudwatchLogs.js

```diff
@@ -0,0 +1,132 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Web-Tier Auto Scaling Group CloudWatch Logs Enabled',
+    category: 'AutoScaling',
+    description: 'Ensures that Web-Tier Auto Scaling Groups are using CloudWatch Logs agent.',
+    more_info: 'EC2 instance available within web-tier Auto Scaling Group (ASG) should use an AWS CloudWatch Logs agent to monitor, store and access log files.',
+    link: 'https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html',
+    recommended_action: 'Update web-tier Auto Scaling Group to use CloudWatch Logs agent',
+    apis: ['AutoScaling:describeAutoScalingGroups', 'AutoScaling:describeLaunchConfigurations', 'STS:getCallerIdentity'],
+    settings: {
+        web_tier_tag_key: {
+            name: 'Auto Scaling Web-Tier Tag Key',
+            description: 'Web-Tier tag key used by Auto Scaling groups to indicate Web-Tier groups',
+            regex: '^.*$',
+            default: 'web_tier'
```

Comment:
@giorod3  Let's change this to an empty string first, which will allow the plugin to exit cleanly without running in the default case _until_ a user enters a setting, which would then activate the plugin.